specs/release-automation/user-triggered-automation-workflow.puml

```diff
@@ -0,0 +1,63 @@
+@startuml
+title Release Automation Process
+
+start
+group RA-1: Trigerring the Process
+    :User passes link to excel dictionary on OSF to use for the release;
+    :User passes OSF personal access token;
+    :User triggers release process;
+end group
+
+group RA-2: Creating the Release Files
+    :Retrieve Excel Dictionary;
+    :Parse metadata from files sheet;
+    :Extract file name and type;
+    :Extract dictionary version from the dictionary file name;
+    :Extract dictionary version from the dictionary summary sheet;
+    if (Version missmatch) then(yes)
+        :Issue error;
+        stop
+    else (no)
+        :Parse the name template string;
```

Comment:
Lines 21 to 42 should go into a while loop right? which iterates through each row in the files sheet?